lib/Tab.js

```diff
@@ -121,9 +126,27 @@ export default class Tab extends Component {
     )
   }
 
+  _renderBadge = () => {
+    const { active, icon, activeIcon, badgeText, badgeSize, badgeStyle, isBadgeVisible } = this.props;
+
+    if (badgeText === undefined && !isBadgeVisible) {
+      return active && activeIcon ? activeIcon : icon;
```

Comment:
_For android compatibility:_ Change to `return null`, so nothing gets rendered if the badge is invisible.

Reply:
done